vms/event/templates/event/edit.html

```diff
@@ -100,76 +100,74 @@
-                                    {% endfor %}
-                                </strong>
-                            </p>
-                        </div>
-                    </div>
+
+            <div id="div_id_country" class="form-group">
+                <label class="col-md-2 control-label">
+                    {% trans "Country" %}
+                </label>
+                <div class="col-md-8">
+                 <select id="select_country" class="form-control" name="country">
+                <option value="0">-- {% trans "Select country" %}
+                {% for country in country_list %}
+                    {% if event.country_id %}
+                     {% if country.name == event.country.name %}
+                         <option selected value="{{country.name}}">{{country.name}}</option>
+                     {% else %}
+                    <option value="{{ country.name }}">{{ country.name }}</option>
+                {% endif %}
                 {% else %}
-                    <div class="form-group">
-                        <label class="col-md-2 control-label">{% trans "Country" %}</label>
-                        <div class="col-md-8">
-                            <input class="form-control" placeholder="{% blocktrans %}Event Country{% endblocktrans %}" type="text" name="country" value="{% if form.country.value %}{{ form.country.value }}{% endif %}">
-                        </div>
-                    </div>
+                <option value="{{ country.name }}">{{ country.name }}</option>
                 {% endif %}
+                {% endfor %}
+            </select>
+            </div>
+            </div>
 
-				{% if form.state.errors %}
-                    <div class="form-group has-error">
-                        <label class="col-md-2 control-label">{% trans "State" %}</label>
-                        <div class="col-md-8">
-                            <input class="form-control" placeholder="{% blocktrans %}Event State{% endblocktrans %}" type="text" name="state" value="{% if form.state.value %}{{ form.state.value }}{% endif %}">
-                            <p class="help-block">
-                                <strong>
-                                    {% for error in form.state.errors %}
-                                        {{ error }}
-                                    {% endfor %}
-                                </strong>
-                            </p>
-                        </div>
-                    </div>
+            <div id="div_id_state" class="form-group">
+                <label class="col-md-2 control-label">
+                    {% trans "State/Province" %}
+                </label>
+                <div class="col-md-8">
+                <select id="select_state" class="form-control" name="state">
+
+                <option value="0">-- {% trans "Select state" %} --</option>
+                {% for state in state_list %}
+                 {% if event.state_id %}
```

Comment:
make indentation uniform here. Somewhere it is too much and somewhere it's two less.